### PR TITLE
Remove chrono backend from spinoso-time

### PR DESCRIPTION
**Pull request by lopopolo**
All of the recent investment in `spinoso-time` has been poured into the
`tz-rs` backend and we've got a high-quality time implementation with
the `tzrs` feature.

The `chrono` backend has atrophied since it was introduced. The API has
diverged from the standard set by the `tz-rs` backend (many APIs which
are fallible in the `tz-rs` backend are infallible in the `chrono`
backend). Full support for the offset API is not something that is
easily achievable with `chrono`.

`chrono` itself is difficult to work with. For example, many APIs panic
on out of range inputs. The API is overly generic which forces
`spinoso-time` to deconstruct a `DateTime` into its raw parts and use
`chrono` types as "views" into the calendar in each accessor method.

`chrono` with time zone support is also a heavy dependency. Removing
this backend results in 13 crates being dropped from `Cargo.lock`.

Especially given the good collaboration we've had with @x-hgg-x (`tz-rs`)
and @Kijewski (`tzdb`, `iana-time-zone`), I don't see a compelling reason
to continue maintaining the `chrono` backend in `spinoso-time`.

### CODE REVIEW

**Comment by lopopolo** [comment]
@b-n I hope you're enjoying your holiday. No rush on this one, but I'd love to get your input and review on this before merging.

**Comment by lopopolo** [comment]
check links job is not cooperating, but no links in markdown sources are broken (or changed?) as far as I know

File: Cargo.toml [hunk 1]
```diff
@@ -133,8 +133,8 @@ core-regexp = ["artichoke-backend/core-regexp"]
 # backend they execute on. The `regex` crate backend remains the default as long
 # as it can parse the given pattern.
 core-regexp-oniguruma = ["core-regexp", "artichoke-backend/core-regexp-oniguruma"]
-# Implement the `Time` core class. This feature adds dependencies on `chrono`
-# and `chrono-tz`.
+# Implement the `Time` core class. This feature adds dependencies on `tz-rs` and
```
Comment (b-n) [inline Cargo.toml:136]: Good catch!

Comment (b-n) [inline Cargo.toml:136]: Good catch!

File: artichoke-backend/src/extn/core/time/mod.rs [hunk 2]
```diff
@@ -14,11 +14,11 @@
 //! ```
 //!
 //! This implementation of `Time` supports the system clock via the
-//! [`chrono`] and [`chrono-tz`] crates.
+//! [`tz-rs`] and [`tzdb`] crates.
```
Comment (b-n) [inline artichoke-backend/src/extn/core/time/mod.rs]: I wonder if just a reference to `spinoso-time` itself is better than the packages that `spinoso-time` relies on. But just a minor nit /shrug

File: spinoso-time/Cargo.toml [hunk 3]
```diff
@@ -14,14 +14,11 @@ keywords = ["artichoke", "date", "datetime", "spinoso", "time"]
 categories = ["date-and-time"]
 
 [features]
-default = ["chrono", "tzrs", "tzrs-local"]
-chrono = ["dep:chrono", "dep:chrono-tz"]
+default = ["tzrs", "tzrs-local"]
```
Comment (b-n) [inline spinoso-time/Cargo.toml:17]: ❤️

File: spinoso-time/README.md [hunk 4]
```diff
@@ -38,32 +37,30 @@ Add this to your `Cargo.toml`:
...
 // Extract the Unix timestamp.
 let timestamp = utc.to_int();
 ```
 
 ## Crate features
 
-This crate supports two backends which are independent of each other. The
+This crate supports several backends which are independent of each other. The
 availability of different backends is controlled by Cargo features, all of which
 are enabled by default:
```
Comment (b-n) [inline spinoso-time/README.md]:
```suggestion
This crate can support several backends, which are designed to be independent
of each other. The availability of different backends is controlled by Cargo
features, all of which are enabled by default:
```

File: spinoso-time/src/lib.rs [hunk 5]
```diff
@@ -34,18 +34,15 @@
...
+//! information, and combines with the [`tz-rs`] crate to generate the time.
 //!
 //! # Crate features
 //!
-//! This crate supports two backends which are independent of each other. The
-//! availability of different backends is controlled by Cargo features, all of
-//! which are enabled by default:
+//! This crate supports several backends which are independent of each other.
+//! The availability of different backends is controlled by Cargo features, all
+//! of which are enabled by default:
```
Comment (b-n) [inline spinoso-time/src/lib.rs]:
```suggestion
//! This crate can support several backends, which are designed to be
//! independent of each other. The availability of different backends is
//! controlled by Cargo features, all of which are enabled by default:
```

File: spinoso-time/src/time/mod.rs [hunk 6]
```diff
@@ -5,25 +5,18 @@
 //! a timezone offset. These timestamps can be used to implement the Ruby `Time`
 //! core class.
 //!
-//! There are two independent backends which can be selected by specifying the
-//! `chrono` or `tzrs` feature:
+//! There are several independent backends which can be selected by specifying
+//! the appropriate feature:
```
Comment (b-n) [inline spinoso-time/src/time/mod.rs:9]:
```suggestion
//! There can be several independent backends which can be selected by
//! specifying the appropriate feature:
```

**Review by b-n — APPROVED** [review]
Something beautiful about so much 🟥. I've put a couple of suggestions around language of "can support" instead of "are" - but they're purely contextual nits, and not super critical IMO.

File: artichoke-backend/src/extn/core/time/mod.rs (hunk 2 above)
Comment (lopopolo) [reply artichoke-backend/src/extn/core/time/mod.rs]: done

File: spinoso-time/README.md (hunk 4 above)
Comment (lopopolo) [reply spinoso-time/README.md]: done locally

File: spinoso-time/src/time/mod.rs (hunk 6 above)
Comment (lopopolo) [reply spinoso-time/src/time/mod.rs:9]: done locally

**Pull request merged**